tests/unit/h_periodic/lms_beat_test.py

```diff
@@ -0,0 +1,4 @@
+# pylint: disable=unused-import
+# This is for some dodgy test coverage
```

Comment:
Copied this from the checkmate equivalent. 

This is probably a little bit better tha ignoring the file if the config file (ie if we add more functions here coverage won't be complete).